modules/ROOT/pages/installation/server.adoc

```diff
@@ -270,4 +270,10 @@ java -jar .\lib\server.jar
 | `OPTOUT_PRODUCT_ANALYTICS`
 | Set to true to opt out of anonymous crash analytics being sent to Neo4j (optional)
 | false
+
+| `grpc.server.security.trustCertCollection`
```

Comment:
I just realized I used the incorrect casing for the analytics stuff here as well, but we don't use camel case in any other the other parameters. Better would be `trust-cert-collection` (yes, I'm aware it's a BE change)